Plk.Blazor.DragDrop.Demo/Pages/TwoZones.razor

```diff
@@ -8,7 +8,7 @@
 
 <div class="bg-light">First Dropzone:</div>
 
-<Dropzone Items="MyFirstList" TItem="TodoItem" OnItemDrop="@((i)=>lastdropped = i)">
+<Dropzone Items="MyFirstList" TItem="TodoItem" OnItemDrop="@((i)=>lastdropped = i)"  AllowsDrag='(d)=>d.Titel == "Item 1"'>
```

Comment:
please create a separate test page - AllowDrag.razor  - because if we put it here, for the viewers of the demo page it is not clear why they can only move "item 1"

Reply:
done!